src/main/java/Lesson4/Task2/Player.java

```diff
@@ -0,0 +1,28 @@
+package Lesson4.Task2;
+
+public class Player implements Playable,Recodable{
+
+    public Player() {
+        super();
+    }
+
+    @Override
+    public void play() {
+        System.out.println("Play");
+    }
+
+    @Override
+    public void record() {
+        System.out.println("Record");
+    }
+
+    @Override
+    public void pause() {
+        System.out.println("Pause");
+    }
+
+    @Override
+    public void stop() {
+        System.out.println("Stop");
+    }
```

Comment:
How the program will know, that is the action is paused or stopped?
You need to have separate pause, stop methods. One of them for recording, the second one for playing